### DISCUSSION TITLE
Plamen Djorev on Soundcloud

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#671] [Show and tell]
https://soundcloud.com/plamen-dzhorev has many songs made with MiniDexed.

Self-described as

> Beginner keyboard player and fan of the synth sound. The music is played on MiniDexed DIY synthesizer, Boss RC-5 loop station and recorded with Audacity. No additional processing is used except normalisation.

I think "beginner" is a gross understatement 👍

**Comment by plamikcho** [comment]
Kind words, thank you! Indeed, it is awesome to be able to make songs with such simple and not expensive gear. Most of my inspiration comes from the excellent performances made by @Banana71 and the expressiveness of MiniDexed. I wish I could play better to release the full potential of MiniDexed - I hope I will improve in the future. For now, I am having fun while "chasing" some nice melodies and trying to play right at least 10 notes in sequence :)

**Reply by probonopd** [reply]
Welcome @plamikcho and thanks for sharing your music!
